README.md

```diff
@@ -121,24 +121,22 @@ Tarball (choose the appropriate version and platform):
 ```
 $ curl -sfLO https://github.com/intelsdi-x/snap/releases/download/0.18.0/snap-0.18.0-linux-amd64.tar.gz
 $ tar xf snap-0.18.0-linux-amd64.tar.gz
-$ cp snapd /usr/local/bin
-$ cp snapctl /usr/local/bin
+$ cp snapteld /usr/local/sbin
+$ cp snaptel /usr/local/bin
 ```
 
-Ubuntu 16.04.1 [snapd package version 2.13+](https://launchpad.net/ubuntu/+source/snapd) installs snapd/snapctl binary in /usr/bin. These executables are not related to snap-telemetry. Running `snapctl` from snapd package will result in the following error message:
+If you installed snap-telemetry package, please make sure `/usr/local/bin:/usr/local/sbin` is part of your $PATH and has higher priority than `/usr/bin:/usr/sbin`, or invoke the snap-telemetry snapteld/snaptel binary using fully qualified path.
```

Comment:
Is this referring to older versions only? Versions post-name change should be fine regardless of where /usr/local/(s)bin is in the path.